.github/workflows/deployment.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     permissions:
       id-token: write
       pages: write
-    if: ${{ github.event_name == 'push' && github.repository == 'DIRACGrid/diracx-web' && github.ref_name == 'main'}}
+    if: ${{ github.event_name == 'push' && github.repository == 'DIRACGrid/diracx-web' }}
```

Comment:
Ideally, `storybook`:
- should be built each time there is a push to `diracx-web` (making sure we don't break it with our changes)
- should be deployed when there is a new release

What do you think?